Makefile

```diff
@@ -250,7 +272,9 @@ start-development-link: build-development-image ## Start linked development envi
 
 	@ echo "Starting development environment (linked)..."
 
+	@ mkdir -p $(CURDIR)/build
```

Comment:
Does the build folder add get removed somewhere? Also, does this fail when the folder already exists (mkdir by default does)?